api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -310,4 +310,61 @@ public void generateVisitNumber_shouldNotThrowOutOfIndexExceptionWhenPreviousQue
 
 		patientQueueingService.generateVisitNumber(location, patient);
 	}
+
+    @Test
+    public void getPatientQueueListBySearchParams_shouldReturnPatientQueueListWithPatientThatMatchesTheSearchStringAndTheGivenStatus() throws Exception {
+
+        PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+
+        Patient patient = Context.getPatientService().getPatient(10000);
+
+        Location location = Context.getLocationService().getLocation(1);
+
+        PatientQueue patientQueue = new PatientQueue();
+        patientQueue.setPatient(patient);
```

Comment:
Why not have presaved values in a dataset file instead of first saving them here?